Vagrantfile

```diff
@@ -269,7 +269,7 @@ Vagrant.configure(2) do |config|
         end
     end
     if ENV["NFS"] then
-        config.vm.synced_folder '.', '/home/vagrant/go/src/github.com/cilium/cilium', type: "nfs", nfs_udp: false
+        config.vm.synced_folder '..', '/home/vagrant/go/src/github.com/cilium', type: "nfs", nfs_udp: false
```

Comment:
>     vagrant: Share parent of cilium/cilium
>
>     Share the parent directory of cilium/cilium. This shares all the
>     cilium repos with one mount, but requires Cilium directory to be named
>     'cilium' (which should usually be the case).

Cannot comment on the commit message, so adding this here :)

We should probably document in the developer docs that the Cilium directory needs to be named `cilium`.

Reply:
Docs added.